ClientServer.go

```diff
@@ -47,7 +46,10 @@ func Server() {
 		// send new string back to client
 		_, sendingerror := conn.Write([]byte(newmessage + "\n"))
```

Comment:
exactly! and its a good practice to reuse variables instead of creating new one.